app/services/analytics.rb

```diff
@@ -157,13 +157,14 @@ def browser_attributes
   IDV_DOC_AUTH_SUBMITTED_IMAGE_UPLOAD_FORM = 'IdV: doc auth image upload form submitted'.freeze
   IDV_DOC_AUTH_SUBMITTED_IMAGE_UPLOAD_VENDOR = 'IdV: doc auth image upload vendor submitted'.freeze
   IDV_DOC_AUTH_SUBMITTED_PII_VALIDATION = 'IdV: doc auth image upload vendor pii validation'.freeze
-  IDV_DOWNLOAD_PERSONAL_KEY = 'IdV: download personal key'.freeze
-  IDV_FINAL = 'IdV: final resolution'.freeze
```

Comment:
`IdV: final resolution` is used in a bunch of IAL2 funnel dashboards (https://github.com/18F/identity-devops/search?q=IdV%3A+final+resolution), WDYT of keeping the old name so we don't break those?

We can rename the constant or add a comment that clarifies it maybe?

Reply:
Part of LG-5221 is to update any existing dashboards as well - there will be a PR there shortly.

Reply:
There's a ~2 week delay on devops deploys... so I think it will be less headache to not rename these

Reply:
Actually, I'll leave it in place at least through the deploy - see a4716c2ac

Reply:
Maybe we could log both temporarily, and add a property to the legacy name so that it's not double-counted in queries?

e.g.

```
filter name = 'IdV: personal key submitted' or (name = 'IdV: final resolution' and properties.event_properties.legacy = 0)
```

Reply:
I had planned to check for either/or, but I like the idea of logging both with a flag as you suggest.

Reply:
Pivoted again - decided to keep `final resolution` but to log it from the end of the Review screen (immediately before the personal key page). This preserves the dashboards :-)